src/ucc/api/ucc.h

```diff
@@ -1651,6 +1669,7 @@ typedef struct ucc_coll_args {
                                              collectives */
     ucc_error_type_t                error_type; /*!< Error type */
     ucc_coll_id_t                   tag; /*!< Used for ordering collectives */
+    void                           *pSync; /*!< pSync array for one-sided collectives */
```

Comment:
what value in ucc_coll_args.mask indicates that psync field is valid?

Reply:
@wfaderhold21  You don't need pWork array for reductions in future?

Reply:
Added the mask and pwrk array.